src/edu/psu/sweng500/team8/gui/CombinationCell.java

```diff
@@ -0,0 +1,244 @@
+package edu.psu.sweng500.team8.gui;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JToggleButton;
+
+import edu.psu.sweng500.team8.coreDataStructures.Cell;
+import edu.psu.sweng500.team8.play.GameSession;
+
+public class CombinationCell extends javax.swing.JPanel {
+	
+	private final static int PENCIL_MARK_SIZE = 18;
+	private final static int PENCIL_MARk_TEXT_SIZE = 15;
+	private final static Color PENCIL_MARK_COLOR = Color.BLACK;
+	private final static Color PENCIL_MARK_BACKGROUND_COLOR = Color.WHITE;
+	private Cell cell;
+	private GameSession gameSession;
+	
+	public CombinationCell(Cell cell, GameSession gameSession) {
+
+		this.cell = cell;
+		this.gameSession = gameSession;
+		
+		setPreferredSize(new Dimension(PENCIL_MARK_SIZE*3, PENCIL_MARK_SIZE*3));
+		setBorder(null);
+		setAlignmentY(Component.TOP_ALIGNMENT);
+		setAlignmentX(Component.LEFT_ALIGNMENT);
+		GridBagLayout gridBagLayout = new GridBagLayout();
+		setLayout(gridBagLayout);
+		
+		//==
+		JToggleButton tglbtnNewToggleButton_1 = new JToggleButton("1");

```

Comment:
I think it is critical that the user can see the pencil marks and the entered values at the same time. Otherwise they will have to constantly toggle back and forth to see anything, which is a poor design. An example is the Sudoku2 App on IOS. Pencil marks are shown in a smaller font in the cell. The cells can fit 6 pencil marks each. When a value is entered, the pencil marks go away.